.github/workflows/automation.yml

```
@@ -23,35 +23,35 @@
     steps:
     - uses: actions/checkout@v4
     - name: ubuntu-qa
-      run: docker build . --file ./ubuntu/Dockerfile.qa --tag ubuntu-qa:$(date +%s)
+      run: docker build . --file ./ubuntu/qa.Dockerfile --tag ubuntu-qa:$(date +%s)
 
   build-ubuntu-latest:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v4
```

yamllint

[error] wrong indentation: expected 6 but found 4

(indentation)